packages/atomic/src/utils/store.ts

```diff
@@ -11,6 +12,7 @@ interface FacetInfo {
 
 interface FacetValueFormat<ValueType> {
   format(facetValue: ValueType): string;
+  content?(facetValue: ValueType): VNode;
```

Comment:
Added a content property instead of extending the formatValue property, the reason is that formatValue is still useful even for rating facet in order to have a relevant formatted title